frontend/app/views/fragments/form/feedback.scala.html

```diff
@@ -64,12 +64,14 @@ <h2 class="fieldset__headline">About you</h2>
             @fragments.form.errorMessage("Please enter your name")
         </div>
 
+        @* email regex taken from http://www.w3.org/TR/html-markup/input.email.html#input.email.attrs.value.single *@
         <div class="form-field">
             <label class="label" for="email">Email address</label>
             <input name="email"
                    id="email"
                    class="input-text"
                    type="email"
+                   pattern="^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$"
```

Comment:
That's not quite true: `type=email` will give users an email specific keyboard but the actual validation part has not-quite-good-enough browser support http://caniuse.com/#search=form%20validation